src/test/java/racingcar/service/TryInputManageTest.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import org.junit.jupiter.api.Test;
+import racingcar.view.InputView;
+
+public class TryInputManageTest {
+    TryInputManage tryInputManage = new TryInputManage();
+    InputView inputView = new InputView();
+
+    @Test
+    void 시도_회수_문자에서_숫자_변환() {
+        String attempt = "2";
+        InputStream in = new ByteArrayInputStream(attempt.getBytes());
+        System.setIn(in);
+        
+        assertThat(tryInputManage.makeTryNumber(inputView))
+                .isEqualTo(Integer.parseInt(attempt));
```

Comment:
이부분 돌려보니까 테스트 통과를 못하더라구요

```suggestion
        assertThat(tryInputManage.makeTryNumber(inputView))
                .isInstanceOf(Integer.class);
```

이런식으로 타입을 비교하는 것이 해당 테스트의 목적에 더 부합한다고 생각되는데 어떻게 생각하시나요? 🙋‍♂️

Reply:
클래스가 변환되니 그러한 방식이 더 보기 좋을 것 같네요!!
저는 코드를 돌리면 테스트가 통과가 되는데 테스트 코드 실행 시 어떠한 오류로 통과가 되지 않았나요?